docs-compose/overwatch_docs/content/DataEngineer/Definitions/071x.md

```diff
@@ -0,0 +1,1005 @@
+| min_num_clusters          | long    | Minimum number of available clusters that will be maintained for this SQL warehouse.                                                                                                                                      | 
+| max_num_clusters          | long    | Maximum number of clusters that the autoscaler will create to handle concurrent querie                                                                                                                                    | 
+| auto_stop_mins            | long    | The amount of time in minutes that a SQL warehouse must be idle (i.e., no RUNNING queries) before it is automatically stopped.                                                                                            | 
+| spot_instance_policy      | string  | Configurations whether the warehouse should use spot instances.                                                                                                                                                           | 
+| enable_photon             | boolean | Configures whether the warehouse should use Photon optimized clusters.                                                                                                                                                    | 
+| channel                   | struct  | This column contains channel details. Some examples - CHANNEL_NAME_UNSPECIFIED, CHANNEL_NAME_PREVIEW, CHANNEL_NAME_CURRENT                                                                                                |
+| enable_serverless_compute | boolean | Flag indicating whether the warehouse should use serverless compute.                                                                                                                                                      |                                                                                                                                                     
+| warehouse_type            | string  | Warehouse type: PRO or CLASSIC                                                                                                                                                                                            |                                                                                                                                                                                           
+| warehouse_state           | string  | State of the warehouse                                                                                                                                                                                                    |                                                                                                                                                                                                   
```

Comment:
if we have these published in DB docs, let's link them

Reply:
Again fetched these values from API docs